src/workerd/tests/BUILD.bazel

```diff
@@ -84,3 +84,34 @@ wd_cc_benchmark(
     srcs = ["bench-fast-api.c++"],
     deps = ["//src/workerd/jsg"],
 )
+
+BENCHMARKS = [
+    ":bench-json",
+    ":bench-mimetype",
+    ":bench-kj-headers",
+    ":bench-api-headers",
+    ":bench-global-scope",
+    ":bench-regex",
+    ":bench-fast-api",
+]
+
+filegroup(
+    name = "all_benchmarks",
+    srcs = BENCHMARKS,
+    visibility = ["//visibility:public"],
+)
```

Comment:
```suggestion
filegroup(
    name = "all_benchmarks",
    srcs = glob(["bench-*"]),
    visibility = ["//visibility:public"],
)
```

Reply:
I don't think this will work as written, globs refer to files not targets. You could build something based on the globs though if you cut off the extension (not sure if that would result in more elegant code).

Reply:
I see. I'll resolve for now.